source/h1_connection.c

```diff
@@ -1095,23 +1057,52 @@ static int s_mark_head_done(struct aws_h1_stream *incoming_stream) {
         aws_h1_decoder_get_header_block(connection->thread_data.incoming_stream_decoder);
 
     if (header_block == AWS_HTTP_HEADER_BLOCK_MAIN) {
-        AWS_LOGF_TRACE(AWS_LS_HTTP_STREAM, "id=%p: Incoming head is done.", (void *)&incoming_stream->base);
+        AWS_LOGF_TRACE(AWS_LS_HTTP_STREAM, "id=%p: Main header block done.", (void *)&incoming_stream->base);
```

Comment:
what's a "main" header block?

Reply:
Invented term for "not 1xx informational header, not trailing header, the normal kind"